gym_jadx/util/PathUtils.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+
+class PathUtils:
```

Comment:
In my initial implementation I did it exactly as you mentioned and it was working. However, when I run JadxEnv
in unit tests (tests/test_jadx_env.py), using paths like 'resources/drawables/main_window.png' caused
FileNotFoundError, since unit test have a different working directory. I couldn't find a workaround, that's why I implemented 
it like this so that I won't have to change the environment code just for testing.